requirements.txt

```diff
@@ -1,5 +1,5 @@
 dogapi==1.2.1
-django>=1.8.14,<1.9
+django<1.9
```

Comment:
Should keep a `>1.8` pin.  It would also be nice not to bound the top unnecessarily, but I guess we don't have tox in place to make sure we test a 1.8 version.